support-frontend/app/controllers/RedemptionController.scala

```diff
@@ -61,7 +61,7 @@ class RedemptionController(
 
   val testUserFromRequest = new TestUserFromRequest(identityService, testUsers)
 
-  def displayForm(redemptionCode: RawRedemptionCode): Action[AnyContent] = (googleAuthAction andThen maybeAuthenticatedAction()).async {
```

Comment:
there is unused code after removing this https://github.com/guardian/support-frontend/blob/32147ee138ad2cf9c731dbba7f92424c1e69f5e0/support-frontend/app/controllers/RedemptionController.scala#L45